sys/parsoid.js

```diff
@@ -888,18 +865,6 @@ module.exports = (options) => {
         // Dynamic resource dependencies, specific to implementation
         resources: [
             {
-                uri: `/{domain}/sys/${options.bucket_type}/parsoid.html`,
-                body: {
-                    revisionRetentionPolicy: {
-                        type: 'latest',
-                        count: 1,
-                        grace_ttl: 86400
-                    },
-                    valueType: 'blob',
-                    version: 1
-                }
-            },
-            {
                 uri: `/{domain}/sys/${options.bucket_type}/parsoid.html-2`,
```

Comment:
Since we don't use `options.bucket_type` any more in the code, we should probably fix this to `key_rev_value`